test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- _(optional)_ I can see you've added the test.md file in gitignore but its still here. This is probably because you did a commit with the test.md file before adding it to the gitignore so I recommend you delete the test.md from the git repository. [Here ](https://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-a-git-repository)is a way you can do that.